src/wolfgame.py

```diff
@@ -6421,7 +6421,7 @@ def fflags(cli, nick, chan, rest):
     else:
         acc, hm = parse_warning_target(params[0])
         flags = params[1]
-        cur_flags = set(var.FLAGS_ACCS[acc] + var.FLAGS[hm])
+        cur_flags = set(var.FLAGS_ACCS[acc.lower()] + var.FLAGS[hm.lower()])
```

Comment:
account names should be lowercased via irc_lower() so they casefold properly. Hostmask is good as-is.